lib/krikri/engine.rb

```diff
@@ -75,6 +75,11 @@ def configure_blacklight!
     initializer :rdf_repository do
       Krikri::Repository =
         RDF::Marmotta.new(Krikri::Settings['marmotta']['base'])
+
+      Krikri::Repository.query_client.instance_eval do
+        @http.keep_alive = 360
```

Comment:
We're using `rails_config`, which is working well enough for us.  You're right that this should be a configuration variable, though.  (Would also be nice if `SPARQL::Client` offered a better way of tweaking the `keep_alive` than `instance_eval`).